src/github.com/getlantern/flashlight/config/bootstrap.go

```diff
@@ -121,7 +134,7 @@ func bootstrapPath(fileName string) (string, error) {
 	}
 	fullPath := filepath.Join(yamldir, fileName)
 	log.Debugf("Opening bootstrap file from: %v", fullPath)
-	return fullPath, nil
+	return yamldir, fullPath, nil
```

Comment:
Yeah in theory the caller should really use filepath.Join, though, which seems like it's kind of asking a bit much